CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 * Remove brackets from show/hide links (PR #448)
 * Add experimental inset text component based on GOV.UK Frontend (PR #449)
 * Add reset styles to document list component (PR #451)
+* Add tests for contextual breadcrumbs (PR #457)
+* Contextual breadcrumbs will show taxon based breadcrumbs if prioritise_taxon_breadcrumbs is true (defaults to false if not passed) (PR #457)
```

Comment:
This could be summarised down to "Add prioritise_taxon_breadcrumbs flag to contextual breadcrumbs (PR #457)". You can give more details about exactly how it works in the PR description if you wish to.